src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php

```diff
@@ -38,7 +40,7 @@ public function getEntries(): ImportMapEntries
 
         $entries = new ImportMapEntries();
         foreach ($importMapConfig ?? [] as $importName => $data) {
-            $validKeys = ['path', 'version', 'type', 'entrypoint', 'url'];
+            $validKeys = ['path', 'version', 'type', 'entrypoint', 'url', 'package_specifier'];
```

Comment:
Supports the rare-use case that you require a package (e.g. `bootstrap`) but want to have its import called something else - `shoestrap` - so you can `import 'shoestrap'`. The `importmap:require` command supports this directly & it's documented.